content/applications/productivity/voip/sales_calls.rst

```diff
@@ -0,0 +1,92 @@
+======================
+ Sales calls with VoIP
+======================
+
+.. |VOIP| replace:: :abbr:`VoIP (Voice over Internet Protocol)`
+
+Using |VOIP| increases call efficiency, reduces costs, and allows a salesperson to access the Odoo
+database while on a call. Instead of relying on a separate phone for sales calls, a salesperson can
+make calls directly from the Odoo database.
+
+This article goes over setting up Odoo |VOIP| for a sales team that already has |VOIP| configured.
+If |VOIP| is **not** configured, refer to the  :doc:`set up VoIP <../voip>` documentation to set up
+|VOIP|.
```

Comment:
I recommend using a :ref: to the specific section here, you'll need to add a custom anchor above the corresponding heading :)
```suggestion
If |VOIP| is **not** configured, refer to the  :ref:`configure VoIP <voip/configure>` documentation
to get started.
```